data/docs/instrumentation/nextjs.mdx

```diff
@@ -0,0 +1,315 @@
+
+const exporterOptions = {
+  url: 'https://ingest.[region].signoz.cloud:443/v1/traces', // use your own data region 
+  headers: { 'SIGNOZ_INGESTION_KEY': 'SIGNOZ_INGESTION_KEY' }, // Use if you are using SigNoz Cloud
+}
+
+const traceExporter = new OTLPTraceExporter(exporterOptions);
+const sdk = new opentelemetry.NodeSDK({
+  traceExporter,
+  instrumentations: [getNodeAutoInstrumentations()],
+  resource: new Resource({
+    [SemanticResourceAttributes.SERVICE_NAME]: 'SigNoz-Nextjs-Example'
+  })
+});
+
+// initialize the SDK and register with the OpenTelemetry API
+// this enables the API to record telemetry
+sdk.start()
+
+// gracefully shut down the SDK on process exit
+process.on('SIGTERM', () => {
+  sdk.shutdown()
+    .then(() => console.log('Tracing terminated'))
+    .catch((error) => console.log('Error terminating tracing', error))
+    .finally(() => process.exit(0));
+});
+```
+
+- `<service_name>` : Name of the service you would like to monitor.
+- `SIGNOZ_INGESTION_KEY` : You can find your ingestion key from SigNoz cloud account details sent on your email.
+
+Depending on the choice of your region for SigNoz cloud, the ingest endpoint will vary according to this table.
+
+| Region | Endpoint |
+| --- | --- |
+| US |	ingest.us.signoz.cloud:443/v1/traces |
+| IN |	ingest.in.signoz.cloud:443/v1/traces |
+| EU | ingest.eu.signoz.cloud:443/v1/traces |
+
+**Step 3.** Once you're done configuring the exporter options, You can validate if your application is sending traces to SigNoz cloud [here](#validating-instrumentation-by-checking-for-traces).
+
+
+#### Send traces via OTel Collector binary
+
+**Step 1.** Install OpenTelemetry Collector binary
+
+OTel Collector binary helps to collect logs, hostmetrics, resource and infra attributes. It is recommended to install Otel Collector binary to collect and send traces to SigNoz cloud. You can correlate signals and have rich contextual data through this way.
+
+You can find instructions to install OTel Collector binary [here](https://signoz.io/docs/tutorial/opentelemetry-binary-usage-in-virtual-machine/) in your VM. 
+
+While creating the `otel-config.yaml` during the installation fo the OTel Collector Binary, you need to enable CORS under the receivers section of the config file. This is needed so that you don't get 
+CORS error which can hinder sending your Traces to SigNoz Cloud. See the code snippet below to understand how you can enable CORS in your config file:
+
+**NOTE:** Make sure to restart your collector after making the config changes
+
+**Step 2.** Install OpenTelemetry packages
+
+```bash
+npm i @opentelemetry/sdk-node
+npm i @opentelemetry/auto-instrumentations-node
+npm i @opentelemetry/exporter-trace-otlp-http
+npm i @opentelemetry/resources
+npm i @opentelemetry/semantic-conventions
+```
+
+**Step 3.** In the `tracing.js` file, remove the headers from the exporterOptions since you're already using the signoz access token in the `otel-config.yml` file.
```

Comment:
If we look carefully, the `tracing.js` flle is almost as same as the previous one except for exporter optons value. IMO, it is better to tell the user about this minute detail rather than asking to create the `tracing.js` file again.